src/hooks/useUndernames/useUndernames.tsx

```diff
@@ -224,6 +224,9 @@ export function useUndernames(id?: ArweaveTransactionID) {
     const undernames = Object.entries(contractState.records);
     for (const [name, record] of undernames) {
       try {
+        if (name === '@') {
+          throw new Error('No undernames to display');
+        }
```

Comment:
no need to error, just filter the main one out above
```
const undernames = Object.entries(contractState.records).filter((key) => key !== '@'));
```

Reply:
added